test/nyc-integration-old.js

```diff
@@ -329,7 +329,53 @@ describe('the nyc cli', function () {
 
         const proc = spawn(process.execPath, args, {
           cwd: fixturesCLI,
-          env: env
+          env
+        })
+
+        let stderr = ''
+        proc.stderr.on('data', function (chunk) {
+          stderr += chunk
+        })
+
+        proc.on('close', function (code) {
+          code.should.equal(1)
+          stderr.should.include('cannot instrument files in place')
+          done()
+        })
+      })
+
+      it('can write files in place with --in-place switch', function (done) {
+        const args = [bin, 'instrument', '--in-place', '--include', '*/file1.js', './instrument-inplace']
```

Comment:
What happens if we run the test twice?  I feel like we need to make a copy of the `instrument-inplace` directory then run the test on the copy.

Reply:
Initially I was thinking of a solution along the lines of making a copy of a test directory then instrumenting in place, then I thought it probably wouldn't matter for the CI infrastructure.  It will of course matter when running in your local env, so I'll make this a bit friendlier.